src/SDKs/SqlManagement/Management.Sql/Microsoft.Azure.Management.Sql.csproj

```diff
@@ -4,7 +4,7 @@
     <PackageId>Microsoft.Azure.Management.Sql</PackageId>
     <Description>Azure SQL Management SDK library</Description>
     <AssemblyName>Microsoft.Azure.Management.Sql</AssemblyName>
-    <VersionPrefix>1.7.0-preview</VersionPrefix>
+    <VersionPrefix>1.8.0-preview</VersionPrefix>
```

Comment:
the latest published version is 1.6.0 (see https://www.nuget.org/packages/Microsoft.Azure.Management.Sql ) so please undo this